ui/DesignSystem/Component/Notification.svelte

```diff
@@ -1,67 +1,69 @@
 <script>
-  import { Icon, Title } from "../Primitive";
+  import { Icon, Text, Title } from "../Primitive";
 
-  export let icon = Icon.Info;
   export let style = null;
-
-  export let level = "info";
+  export let variant = "info"; // info | error
```

Comment:
You will have to change the prop for `Notification` in `NotificationFaucet` as well.

Reply:
Ok, took a stab at it. I also changed  ` level: message.level` to `variant: message.variant` I don't know if that makes sense or if the message is referenced somewhere else as well.